src/recipe/s3.service.ts

```diff
@@ -48,4 +49,13 @@ export class S3Service {
     const command = new PutObjectCommand(params);
     await this.s3Client.send(command);
   }
+
+  async deleteFileFromS3(bucket: string, key: string): Promise<void> {
```

Comment:
you have only one bucket. All of the s3 methods does not have to take bucket as a param